### PR TITLE
Add Javascript object for popup window

### PR DESCRIPTION
**Pull request by Kanyorok**
Add a popup window for the portfolio website version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -357,6 +383,29 @@ body {
...
+    bottom: 0;
+    overflow: auto;
+    background: rgba(0, 0, 0, 0.7);
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-bottom: 39px;
+    padding-top: 60px;
+    z-index: 2;
+    visibility: hidden;
+  }
```
Comment (taldr27) [inline style.css]: Amazing job on the mobile popup! I have **a couple of small suggestions to further improve it and bring it closer to the provided** [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=6Kf7cezM539HxgRb-0):

- [ ] Please double-check that the popup is correctly centered on all devices like 375x667 or 414x896. It seems that the fixed `padding-top` and `padding-bottom` you added might **be causing some alignment issues**. I recommend using percentages or considering **centering it with Flexbox** for a better result. 🔍

- [ ] Additionally, please ensure that you're using **the correct image from the template**. It appears that you're currently using a different one. Please make the necessary adjustments to match the template more closely. 🙋‍♂️

- [ ] **[OPTIONAL]** You could consider adding a bit more example text to your paragraph to make it look closer to the template. 😉

  _Watch the next video to gain a better understanding and see a possible solution for your problem._ 🎬

  [5_19_2023_4-36-34 PM.webm](https://github.com/Kanyorok/Portfolio_Setup/assets/107952724/ce60e799-0399-41ba-9508-9c0306ee421a)

File: style.css [hunk 2]
```diff
@@ -357,6 +383,29 @@ body {
...
+
+  .popup_content {
+    display: flex;
+    flex-direction: column;
+    background: #fff;
+    border-radius: 2px;
+    padding: 16px;
+  }
```
Comment (taldr27) [inline style.css:407]: Well done on implementing the popup in the desktop version! To **bring it even closer to the [provided template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=6Kf7cezM539HxgRb-0)**, I have a few styling suggestions:

- [ ] Please verify that you are using the correct size of the image in your popup. Additionally, **consider replacing it with the image provided in the template for better accuracy**. 👌

- [ ] Ensure that your buttons are aligned correctly, following the design specifications in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=6Kf7cezM539HxgRb-0). They should be positioned at the bottom left. **This adjustment will enhance the overall professionalism and craftsmanship of your popup.** 😉

  _By making these small refinements, you'll achieve a more faithful representation of the template and create a polished and visually appealing desktop popup. Keep up the great work!_ 🎯

  ### Your design:
  image
  ### Figma design:
  image
  
    **_Just a friendly reminder that adhering to the template is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md). Thanks for considering these suggestions, and keep up the great work!_**
image

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello team @Kanyorok, @khanh-devos, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Populating the popup dynamically.
✅ Popup functionality working fine.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Kanyorok, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Popup functionality working fine.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**